README.md

```diff
@@ -407,6 +407,7 @@ Table of Contents
 
 ## Security and PKI
 
+  * [Acra](https://marketplace.digitalocean.com/apps/acra) - SQL database protection suite: field-level encryption, masking, tokenization, available as database proxy or API service.
```

Comment:
Unfortunately no, its not a SaaS in its free tier.  You are required to pay for instances on DO in order to use it and then once running on those instances its effectively self hosted.

Other systems that manages infrastructure that you host and that are listed here offer a SaaS component that actively manage those hosted machines, for example monitoring tools that can monitor 10 nodes, or kubernetes deployment systems that can deploy to on k8s.  

This doesnt appear to have a SaaS component once deployed off DO marketplace.

Reply:
Okay, thank you